libs/common/src/lib/deep-freeze.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { deepFreeze } from './deep-freeze';
+
+// Created a cool set of unit tests with ChatGPT: https://chat.openai.com/share/f2a078c9-e996-4d4e-8c4c-725ca2e9cafd
```

Comment:
Perhaps this comment can be removed?

Reply:
I prefer to be transparent about copy-pasted stuff from ChatGPT or stack-overflow.

The comment text I adjusted a little bit now.